libs/product/src/facades/product/product.facade.ts

```diff
@@ -17,23 +17,25 @@ import { DaffProductFacadeInterface } from './product-facade.interface';
 })
 export class DaffProductFacade<T extends DaffProduct = DaffProduct> implements DaffProductFacadeInterface<T> {
   /**
-   * The loading state of the currently selected product.
+   * The loading state of the product.
    */
   loading$: Observable<boolean>;
   /**
    * The currently selected product.
+	 * @deprecate use getProduct instead.
    */
   product$: Observable<T>;
 
-  constructor(private store: Store<DaffProductReducersState<T>>) {
-		const {
-			selectSelectedProductLoadingState,
-			selectSelectedProduct
-		} = getDaffProductSelectors<T>();
+	selectors = getDaffProductSelectors<T>();
```

Comment:
I think this field should be `private` or `readonly`.

Reply:
done